### PR TITLE
generate_tokens(readline) must return bytes

### PR DESCRIPTION
**Pull request by srittau**

### CODE REVIEW

File: stdlib/3/tokenize.pyi [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Untokenizer:
 def untokenize(iterable: Iterable[_Token]) -> Any: ...
 def detect_encoding(readline: Callable[[], bytes]) -> Tuple[str, Sequence[bytes]]: ...
 def tokenize(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]: ...
-def generate_tokens(readline: Callable[[], str]) -> Generator[TokenInfo, None, None]: ...
+def generate_tokens(readline: Callable[[], bytes]) -> Generator[TokenInfo, None, None]: ...  # undocumented
```
Comment (blueyed) [inline stdlib/3/tokenize.pyi]: That's incorrect, isn't it?
https://github.com/python/cpython/blob/825358ab80d0e17e287c1e36a91dece0a97cb219/Lib/tokenize.py#L612-L618

Comment (JelleZijlstra) [reply stdlib/3/tokenize.pyi]: Yes, confirmed in shell on 3.6. @srittau what was the basis for your change?

Comment (blueyed) [reply stdlib/3/tokenize.pyi]: I've noticed that there are difference with py27 in this regard, so maybe it got confused?

Comment (srittau) [reply stdlib/3/tokenize.pyi]: No idea why I thought readline() should return bytes or why I even stumbled over this. I probably misread `_tokenize()`.

**Pull request merged**